quizcreator/src/components/navbar.js

```diff
@@ -0,0 +1,30 @@
+import Image from 'next/image';
+import React from 'react'
+import { Inter } from 'next/font/google'
+const inter = Inter({ subsets: ['latin'] })
+
+const Navbar = ()=> {
+  return (
+ <>
+ <div className=''>
+  <div className='flex flex-row justify-between'>
+      <Image
+      src="/logo_2 1.svg"
```

Comment:
use a better name, 
you can use favicon from here https://github.com/avantifellows/portal-frontend/tree/main/src/assets/images